pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -65,7 +65,7 @@ const (
 	dedupChain = utilebtables.Chain("KUBE-DEDUP")
 
 	// defaultIPAMDir is the default location for the checkpoint files stored by host-local ipam
-	// https://github.com/containernetworking/cni/tree/master/plugins/ipam/host-local#backends
+	// https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md
 	defaultIPAMDir = "/var/lib/cni/networks"
```

Comment:
It appears that nothing uses this constant any more; the code that used it was removed in #62212. So can you just remove this as well?

Reply:
Good catch. Would that be ok to do in this PR or should I open a separate one with a better summary and title?

Reply:
I created https://github.com/kubernetes/kubernetes/pull/81617 for this